xmtp_mls/src/subscriptions/stream_all.rs

```diff
@@ -105,6 +120,18 @@ where
         let mut this = self.as_mut().project();
 
         if let Ready(msg) = this.messages.as_mut().poll_next(cx) {
+            if let Some(Ok(msg)) = &msg {
+                if self.sync_groups.contains(&msg.group_id) {
+                    let _ = self
+                        .client
+                        .local_events()
+                        .send(LocalEvents::SyncWorkerEvent(
+                            SyncWorkerEvent::NewSyncGroupMsg,
```

Comment:
looks good, jw about this part:
when this is sent, what happens in the sync worker? Would it be useful to send the message itself in the channel, or how does the worker get the actual message